src/lib/components/Search.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import { createEventDispatcher } from "svelte";
+
+    let searchTerm = '';
+
+    const dispatch = createEventDispatcher();
+
+    function onSubmit(){
+        dispatch("search", {searchTerm})
+    }
+
+</script>
+
+<form on:submit|preventDefault={onSubmit} class="p-4 space-y-4">
+	<label for="search" />
+	<input
+		type="text"
+		bind:value={searchTerm}
+		placeholder="Search by keyword"
+		id="search"
+		name="search"
+		class="p-2 rounded-lg border-2 border-green-500 dark:border-green-700 text-green-700 dark:text-green-500"
+	/>
+	<button
+		type="submit"
+		class="p-2 rounded-lg bg-green-700 text-white dark:bg-green-900/75"
+	>
+		Search
+	</button>
```

Comment:
should be able to use the component here for a single source of truth on all these colour classes for light/dark/hover


```suggestion
	 <ButtonLinks
                    type="submit"
                    version="filled"
                    color="green"
                >
                    <span slot="label">Search</span>
                </ButtonLinks>
```

Reply:
Using ButtonLinks component for light/dark/hover